crates/swc_ecma_parser/src/parser/typescript.rs

```diff
@@ -670,10 +667,7 @@ impl<I: Tokens> Parser<I> {
                 TsEnumMemberId::Str(Str {
                     span,
                     value: v.to_string().into(),
-                    has_escape: false,
-                    kind: StrKind::Normal {
-                        contains_quote: false,
-                    },
+                    raw: Some(format!("\"{}\"", v).into()),
```

Comment:
Is having this better?

Reply:
@kdy1 Hm, do you mean

> You can manually use String for better performance, but I don' think it will have a huge didfference.
(I mean String::push & String::push_str)

Just create `String::new()`? I missed that because it is recovery mode

Reply:
I think in real code this will not happen... but I can refactor to using `String`